scripts/monitor-log-files.sh

```
@@ -0,0 +1,66 @@
+
+    # Process history log silently
+    total_entries=$(wc -l < "$LOG_HISTORY_FILE")
+    error_count=$(grep -ci "error" "$LOG_HISTORY_FILE")
+    grep -i "error" "$LOG_HISTORY_FILE" | tail -n 5 > "$OUTPUT_DIR/history_report.txt"
+
+    # Process system error logs silently
+    sudo journalctl -p err --no-pager | tail -n 5 > "$OUTPUT_DIR/system_error_report.txt"
+
+    # Create backups (overwrite existing ones)
+    cp "$LOG_HISTORY_FILE" "$OUTPUT_DIR/history.backup"
+    sudo journalctl -p err --no-pager > "$OUTPUT_DIR/system_errors.backup"
```

Shellcheck (0.10.0)

[warning] sudo doesn't affect redirects. Use ..| sudo tee file

(SC2024)